### PR TITLE
REMOVE nodeenv development

### PR DESCRIPTION
**Pull request by ndelangen**
fixes: https://github.com/chromaui/chromatic-cli/issues/68

### CODE REVIEW

**Comment by tmeasday** [comment]
What about the `BROWSER` bit? I don't know why that's there either.

**Comment by tmeasday** [comment]
The commit where it cate from is super old: https://github.com/chromaui/chromatic/commit/e1a9ba09283ff96a6c9b369e855db5c3bd41efba

**Comment by ndelangen** [comment]
The NODE_ENV=TEST was to make babel parse the example storybook correctly 😖 

Removing it caused the example to fail on require.context

I was able to make the example work by removing the babel stuff for webpack's require.context, and migrating to tri-config.

**Comment by ndelangen** [comment]
I merged in the better error handling, because it actually helped my find & test the problem.

I could remove it from this branch again if you really want @tmeasday or I could close the other PR, and consider it 1 cleanup PR. WDYT?

**Comment by tmeasday** [comment]
Hold on a sec.

**Comment by ndelangen** [comment]
I cleaned the PR up @tmeasday

File: .babelrc.js [hunk 1]
```diff
@@ -4,9 +4,6 @@ module.exports = {
       presets: [
         ['@babel/preset-env', { targets: { node: 'current' } }],
       ],
-      plugins: [
-        'require-context-hook'
-      ],
```
Comment (ndelangen) [inline .babelrc.js]: The environment variable NODE_ENV=test essentially caused the static storybook to be build with this babel config.

**Comment by tmeasday** [comment]
Thanks it looks reasonable. I'm just worried that you had to do this. Does it mean that Chromatic doesn't work any more with non-`main.js` storybooks?

**Comment by ndelangen** [comment]
NO AFAICS it should work without / should work with the old config.js just fine @tmeasday

**Comment by tmeasday** [comment]
So why did you need to do it in our test storybook? Can you explain what happened? Is the `BROWSER` var needed?

**Comment by ndelangen** [comment]
@tmeasday I think build-storybook uses the babel config defined in the root. It's always done that, but because NODE_END was set to 'development' and in the root there's no babel config for that env, it wouldn't override anything.

Removing NODE_ENV='development' made it act as NODE_ENV='test' which DOES match a env specified in the root babel config, and thus it's applied.

Turns out if we add `NODE_ENV=test` here:
https://github.com/chromaui/chromatic-cli/blob/fb07c9314ff587ad2623ca79d9b9fe92cf7d0072/bin/register.js#L3

IMHO that should be `production` WDYT?

**Comment by ndelangen** [comment]
We can't know exactly what impact a NODE_ENV change will have on the users config/code-output. Perhaps it's safer to release this as a major?

**Comment by ndelangen** [comment]
AFACS there's no usage of process.env.BROWSER whatsoever. Removing seems to do absolutely nothing.

**Comment by tmeasday** [comment]
OK, setting `NODE_ENV=test` there seems not correct either. Do you know why we did that? Was it for the sake of dotenv?

> We can't know exactly what impact a NODE_ENV change will have on the users config/code-output. Perhaps it's safer to release this as a major?

You are probably right.

**Comment by ndelangen** [comment]
I'll delete this code and that makes this a major release

```
process.env.NODE_ENV = process.env.NODE_ENV || 'test'; 
```

**Pull request merged**